pkg/lib/cockpit-components-dialog.jsx

```diff
@@ -339,8 +339,8 @@ export function show_modal_dialog(props, footerProps) {
 }
 
 export function apply_modal_dialog(event) {
-    const dialog = event && event.target && event.target.closest("[role=dialog]");
-    const button = dialog && dialog.querySelector("button.apply");
+    const dialog = event?.target?.closest("[role=dialog]");
+    const button = dialog?.querySelector("button.apply");
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18448-20230312-091152-46c0aa82-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)